translator.py

```diff
@@ -0,0 +1,129 @@
+import os
+word = str("")
+final_word = str(input())
+while final_word != word:
+    Trigger = float(input("Enter your Trigger: "))
+    Stop = float(input("Enter your Stop: "))
+    Executed_trigger = float(input("When did Trigger execute?"))
+    Trade_Type = input("Is the trade Long or Short? ")
+
+    #Code for Long trades:
+
+    if Trade_Type == "Long":
+        Shares = (abs(50 / (Trigger - Stop)))
+        one_one_partial = input("Did you reach your 1:1 target of " + str(((Trigger - Stop) + Trigger)) + str("?"))
+
+        #first "if" checks if any of the trade succeeded
+
+        if  one_one_partial == "no":
+            Executed_stop = float(input("When did Stop execute?"))
+            loss = abs(Executed_trigger - Executed_stop) * Shares
+            print("You lost " + str(loss) + "$ on all trades")
+        elif one_one_partial == "yes":
+            one_one_partial_profit = "%.2f" %((((Trigger - Stop) + Trigger) - Executed_trigger) * 0.75 * Shares)
+            one_one_full_profit = "%.2f" %((((Trigger - Stop) + Trigger) - Executed_trigger) * Shares)
+            remainder_one_one = float(input("Where did you cover remainder of 1:1 trade: "))
+            remainder_one_one_profit = "%.2f" %((remainder_one_one - Executed_trigger) * 0.25 * Shares)
+
+            #This "if" checks if a 2:1 type of trade succeeded
```

Comment:
Comment should be right above the section you document, without any additional spaces.